### PR TITLE
Adds GH Actions pipeline, postgres-matrix and packaging_ignore file

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

File: .github/workflows/build-package.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+jobs:
+  build_package:
+    name: Build package
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - el/8
```
Comment (hanefi) [inline .github/workflows/build-package.yml:27]: Do we actually use el/8 or el/7 ?

File: .github/workflows/build-package.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      - "**"
+
+  workflow_dispatch:
+
+jobs:
+  build_package:
+    name: Build package
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```
Comment (hanefi) [inline .github/workflows/build-package.yml:23]: Why do we not fail fast? Any particular reason?

File: packaging_ignore.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  - ".* warning: overriding recipe for target 'check'"
+  - "dpkg-genchanges: warning: the current version .* is earlier than the previous one .*"
+  - ".*: warning: '__s' may be used uninitialized in this function \\[-Wmaybe-uninitialized\\].*"
+  - ".* unused substitution variable \\$\\{misc:Depends\\}.*"
+  - ".*: substitution variable \\$\\{postgresql:Depends\\} unused, but is defined.*"
+  - ".* unused substitution variable \\$\\{postgresql:Depends\\}.*"
+
+debian:
+  - ".*: W: invalid-license Commercial"
+  - ".*: W: no-documentation"
```
Comment (hanefi) [inline packaging_ignore.yml:19]: Do we want to keep Debian specific rules here as well?

I am ok with keeping those here for consistency. There is no harm in doing this I think.

File: .github/workflows/build-package.yml [hunk 4]
```diff
@@ -0,0 +1,61 @@
+name: Build Package
```
Comment (hanefi) [inline .github/workflows/build-package.yml]: can we give this a better name that also contains azure gdpr in it?

File: .github/workflows/build-package.yml (hunk 1 above)
Comment (gurkanindibay) [reply .github/workflows/build-package.yml:27]: el/7 also exist in travis file. So I added el/7 as well

File: .github/workflows/build-package.yml (hunk 2 above)
Comment (gurkanindibay) [reply .github/workflows/build-package.yml:23]: Yes matrix jobs are independent from each other. So If a package job fails for an os version, I want other jobs continue. For some cases, there may be some os specific problems and I don't want these spesific problems to block other os jobs

File: packaging_ignore.yml (hunk 3 above)
Comment (gurkanindibay) [reply packaging_ignore.yml:19]: I just want it consistent as much as possible as you mentioned

File: .github/workflows/build-package.yml (hunk 4 above)
Comment (gurkanindibay) [reply .github/workflows/build-package.yml]: Changed  and gave a more descriptive name

**Review by hanefi — APPROVED** [review]

**Pull request merged**